### PR TITLE
Italian Localization

### PR DESCRIPTION
**Pull request by gotamm**
Hi @massivefermion,

Following what's was done for other languages this PR adds the support for Italian language.
Hoping that I have done something you appreciate, I remain open to any suggestions or requests for changes.

### CODE REVIEW

File: src/blah/it/address.gleam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  get_random_item(address.cities)
+}
+
+pub fn province() {
+  get_random_item(address.provinces)
+}
+
+pub fn street() {
+  let street = get_random_item(address.streets)
+  // most of Italian street names starts with "rue"
```
Comment (massivefermion) [inline src/blah/it/address.gleam]: I don't understand. The prefix you're attaching is `Via`, but you say `rue` in the comment!

Comment (gotamm) [reply src/blah/it/address.gleam]: I'm really sorry for the late response. 
You're totally right, it was my mistake

**Comment by massivefermion** [comment]
I totally forgot about this.
It seems like the formatting is off in some places. Please update the formatting using the latest gleam version.

**Comment by massivefermion** [comment]
thanks

**Comment by massivefermion** [comment]
I'd like to thank you in the discord server so I'd appreciate it if you DM me there.

**Pull request merged**